kubernetes/config/frontend.yaml

```diff
@@ -14,7 +13,10 @@ data:
         '': { endpoint: { address: 'scheduler:8982' } },
       },
 
+      allowAcUpdatesForInstanceNamePrefixes: [''],
```

Comment:
I think it's good to document that it may be useful to set this option, but I don't think we should have it enabled by default. There are many cases in which people want to do pure remote execution, not allowing clients to poison the Action Cache of the cluster.